From: Modestas Vainius <modestas@vainius.eu>
Subject: Prefer *-qt3 binaries to generic names which might come from Qt4
 *-qt3 names are Debian specific. Nevertheless, the patch could be applied
 upstream.
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=538864

--- a/Modules/FindQt3.cmake
+++ b/Modules/FindQt3.cmake
@@ -138,7 +138,7 @@
 
 # qt 3 should prefer QTDIR over the PATH
 FIND_PROGRAM(QT_MOC_EXECUTABLE
-  NAMES moc moc-qt3
+  NAMES moc-qt3 moc
   HINTS
   $ENV{QTDIR}/bin
   PATHS
@@ -160,7 +160,8 @@
 ENDIF(QT_MOC_EXECUTABLE)
 
 # qt 3 should prefer QTDIR over the PATH
-FIND_PROGRAM(QT_UIC_EXECUTABLE uic
+FIND_PROGRAM(QT_UIC_EXECUTABLE
+  NAMES uic-qt3 uic
   HINTS
   $ENV{QTDIR}/bin
   PATHS
